planning/behavior_velocity_planner/autoware_behavior_velocity_intersection_module/src/object_manager.hpp

```diff
@@ -126,17 +126,16 @@ class ObjectInfo
    * @brief update predicted_object_, attention_lanelet, stopline, dist_to_stopline
    */
   void initialize(
-    const autoware_perception_msgs::msg::PredictedObject & predicted_object,
-    std::optional<lanelet::ConstLanelet> attention_lanelet_opt,
-    std::optional<lanelet::ConstLineString3d> stopline_opt);
+    const autoware_perception_msgs::msg::PredictedObject & object,
+    std::optional<lanelet::ConstLanelet> attention_lanelet_opt_,
+    std::optional<lanelet::ConstLineString3d> stopline_opt_);
```

Comment:
Using `hoge_` naming for parameters/arguments is not reccomended, since they are for global variables.

Reply:
What does it mean when _ is used in the original function argument?

```
void ObjectInfo::initialize(
  const autoware_perception_msgs::msg::PredictedObject & object,
  std::optional<lanelet::ConstLanelet> attention_lanelet_opt_,
  std::optional<lanelet::ConstLineString3d> stopline_opt_)
```

Reply:
@kyoichi-sugahara @soblin @shmpwk 
What was the original intention of using `hoge_` names for function parameters?

Reply:
@kyoichi-sugahara  @soblin  @shmpwk 
Hi

Reply:
@kobayu858 Can you rename
- `ObjectManager::attention_lanelet_opt` to `ObjectManager::attention_lanelet_opt_`
- `ObjectManager::stopline_opt` to `ObjectManager::stopline_opt_`
- `ObjectManager::dist_to_stopline_opt` to `ObjectManager::dist_to_stopline_opt_`

Reply:
@soblin 
Added `_` to member variables and removed `_` from arguments.